.ci/install_cc_tests.sh

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2017 Intel Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#!/bin/bash
+
+set -e
+
+echo "Retrieve cc-tests repository"
+go get -d github.com/clearcontainers/tests
+
+echo "Retrieve cc-tests dependencies"
```

Comment:
In any case, this should be part of the `tests` repo. proxy testing will need it as well for instance.